cookbooks/aws-parallelcluster-slurm/templates/default/slurm/login_nodes_daemon_service/login_nodes_daemon_script.sh.erb

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This daemon and the script it calls does not echo any information about its execution. Let's think about useful info to be logged and we will make supervisord log into a dedicated log file